frontend/src/ReviewPage.js

```diff
@@ -132,12 +132,13 @@ class ReviewPage extends Component {
   }
 
   showRowHistory(rowCode) {
-    this.setState({ rowCode }, () =>
-      window.scrollTo({
-        behavior: 'smooth',
-        top: this.myRef.current.offsetTop,
-      })
-    )
+    this.setState({ rowCode }, () => {
+      if (rowCode)
+        window.scrollTo({
+          behavior: 'smooth',
+          top: this.myRef.current.offsetTop,
+        })
```

Comment:
if prettier formats this into two lines like this, I would prefer adding the brackets back